ArjunaJTS/idl/idlj/pom.xml

```diff
@@ -41,6 +42,239 @@
       </plugin>
 
       <plugin>
+        <artifactId>maven-antrun-plugin</artifactId>
+        <version>1.7</version>
+        <executions>
+          <execution>
+            <id>generate-sources</id>
+            <phase>generate-sources</phase>
+            <goals>
+              <goal>run</goal>
+            </goals>
+            <configuration>
+              <tasks name="Generate idl stubs">
+                <java classname="com.sun.tools.corba.se.idl.toJavaPortable.Compile">
+
+                  <arg value="-i"/>
+                  <arg value="${project.build.directory}/../../src/main/idl"/>
+                  <arg value="-td"/>
+                  <!-- TODO figure out why the compiler does not pick up these sources
```

Comment:
If you mean automatically its because maven doesn't know anything about that folder unless you tell it. I think you can use something called build-helper-plugin to add source paths.

Maybe it was a "memo-to-self" in which case sorry for intruding :) If not, please can you add a little more context?